.github/workflows/pr.yml

```diff
@@ -15,6 +15,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Set Ruby Version
+        uses: ruby/setup-ruby@v1
```

Comment:
seems like .ruby-version is being ignored, for diag could try forcing it here: 
with:
        ruby-version: 2.7.3

(as per https://github.com/ruby/setup-ruby#usage)